bin/dev/kube-ready-state-check.sh

```diff
@@ -87,38 +91,40 @@ fi
 # kube-dns shows 4/4 ready
 if having_category kube ; then
     kubectl get pods --namespace=kube-system --selector k8s-app=kube-dns | grep -Eq '([0-9])/\1 *Running'
-    status "kube-dns should shows 4/4 ready"
+    status "kube-dns should show 4/4 ready"
 fi
 
 # ntp is installed and running
 if having_category api kube node ; then
-    systemctl is-active ntpd >& /dev/null || systemctl is-active systemd-timesyncd >& /dev/null
-    status "ntp or systemd-timesyncd must be installed and active"
+    pgrep -x ntpd >& /dev/null || pgrep -x chronyd >& /dev/null || systemctl is-active systemd-timesyncd >& /dev/null
+    status "An ntp daemon or systemd-timesyncd must be installed and active"
 fi
 
 # At least one storage class exists in K8s
 if having_category kube ; then
-    test $(kubectl get storageclasses |& wc -l) -gt 1
+    test "$(kubectl get storageclasses |& wc -l)" -gt 1
     status "A storage class should exist in K8s"
 fi
 
 # privileged pods are enabled in K8s
 if having_category api ; then
-    kube_apiserver=$(systemctl status kube-apiserver -l | grep "/usr/bin/hyperkube apiserver" )
+    kube_apiserver=$(pgrep -ax hyperkube | grep " apiserver " )
     [[ $kube_apiserver == *"--allow-privileged"* ]]
     status "Privileged must be enabled in 'kube-apiserver'"
 fi
 
 if having_category node ; then
-    kubelet=$(systemctl status kubelet -l | grep "/usr/bin/hyperkube kubelet" )
+    kubelet=$(pgrep -ax hyperkube | grep " kubelet " )
     [[ $kubelet == *"--allow-privileged"* ]]
     status "Privileged must be enabled in 'kubelet'"
 fi
 
 # override tasks infinity in systemd configuration
 if having_category node ; then
-    systemctl cat containerd | grep -wq "TasksMax=infinity"
-    status "TasksMax must be set to infinity"
+    if has_command isystemctl ; then
```

Comment:
Yes, definitely a typo. Fixed in 24282002, I had to rebase and force push because of a conflict with the new tiller checks.